keypoints/include/pcl/keypoints/harris_6d.h

```diff
@@ -126,7 +126,7 @@ namespace pcl
       void detectKeypoints (PointCloudOut &output);
       void responseTomasi (PointCloudOut &output) const;
       void refineCorners (PointCloudOut &corners) const;
-      void calculateCombinedCovar (const std::vector<int>& neighbors, float* coefficients) const;
+      void calculateCombinedCovar (const pcl::Indices& neighbors, float* coefficients) const;
```

Comment:
I hope `neighbors` is a vector of indices... (didn't check src code, just pointing out a possible concern)

Reply:
Yes, it is a vector of indices. [Here](https://github.com/PointCloudLibrary/pcl/blob/master/keypoints/include/pcl/keypoints/impl/harris_6d.hpp#L75) is the implementation, where the elements of neighbors are used to access a normals cloud. [And here](https://github.com/PointCloudLibrary/pcl/blob/master/keypoints/include/pcl/keypoints/impl/harris_6d.hpp#L288), `calculateCombinedCovar` is called with the resulting indices of a `radiusSearch`.